client/src/app/templates/new/templates-new.component.ts

```diff
@@ -1,84 +1,116 @@
 import { Component } from '@angular/core';
-import { FormArray, FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { AbstractControl, FormArray, FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { Template } from '../../_models/index';
+import { TemplateService } from '../../_services/template.service';
+import { ToasterService } from '../../_services/toaster.service';
+import { Router } from '@angular/router';
 
 @Component({
     templateUrl: './templates-new.component.html',
 })
 
 export class TemplateNewComponent {
-    name: String;
-    template: Template = new Template();
 
-    startAt: Date;
-    endAt: Date;
-    selectedDay: Number;
-    nbMax: Number;
-
-    services: Array<{
-        nbMax: Number,
-        startAt: {
-            day: Number,
-            hours: Number,
-            minutes: Number
-        },
-        endAt: {
-            day: Number,
-            hours: Number,
-            minutes: Number
-        }
-    }>;
-
-    serviceForm: FormGroup;
-    templateForm: FormGroup;
-    formArray: FormArray;
+    templateNameFormGroup: FormGroup;
+    servicesFormArray: FormArray;
+    generalFormArray: FormArray;
+    generalFormGroup: FormGroup;
 
     WEEK_DAY = [
-        { id: '1', value: 'Lundi' },
-        { id: '2', value: 'Mardi' },
-        { id: '3', value: 'Mercredi' },
-        { id: '4', value: 'Jeudi' },
-        { id: '5', value: 'Vendredi' },
-        { id: '6', value: 'Samedi' },
-        { id: '7', value: 'Dimanche' } ];
+        { id: '0', value: 'Lundi' },
+        { id: '1', value: 'Mardi' },
+        { id: '2', value: 'Mercredi' },
+        { id: '3', value: 'Jeudi' },
+        { id: '4', value: 'Vendredi' },
+        { id: '5', value: 'Samedi' },
+        { id: '6', value: 'Dimanche' } ];
```

Comment:
Are you sur about using 0 for Monday? I think that most of libs use 0 for Sunday?

Reply:
Hum...
Let's change that !

Reply:
![image](https://user-images.githubusercontent.com/29222996/40414905-d5330ae4-5e79-11e8-90f7-d292f07cec6b.png)

So we don't change it :)

Reply:
So you should revert the changes 😸

Reply:
Yes :) I've changed the BACK template model (get & set)